### PR TITLE
[Release] Bump version to alpha 9

### PR DESCRIPTION
**Pull request by elliotcmorris**
Shouldn't merge until traitgen alpha 9 is released.
https://github.com/OpenAssetIO/OpenAssetIO-TraitGen/pull/75

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
 Release Notes
 =============
 
+v1.0.0-alpha.9
+---------------
+
+### Improvements
+
+- Update `openassetio-traitgen` to `v1.0.0a9` to bring in new traitview
```
Comment (feltech) [inline RELEASE_NOTES.md:9]: [consider] "new traitview features" could be a bit misleading, i.e. what is "traitview".  Perhaps explicitly say moving from exceptions in C++ to std::optional.

**Review by feltech — APPROVED** [review]

File: RELEASE_NOTES.md (hunk 1 above)
Comment (elliotcmorris) [reply RELEASE_NOTES.md:9]: Fair point. Do we have a word for what our generated types are? Just "Generated view types"?

Comment (elliotcmorris) [reply RELEASE_NOTES.md]: [abfe8de](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/86/commits/abfe8defc184e253446c467921a6cbf7affe977d)

Comment (feltech) [reply RELEASE_NOTES.md]: I'd say "generated trait view classes", a bit of a mouthful.  Though what you've written LGTM.

**Review by feltech — APPROVED** [review]

**Pull request merged**